### PR TITLE
draft for split permissions

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Purpose

Adding informations on split permissions. 

## Merge requirements

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] New features docs

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

### CODE REVIEW

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 1]
```diff
@@ -11,91 +11,89 @@ helpdocs_is_private: false
...
-
-1. If you're sure you want to archive the flag, type the name or ID of the flag in the field provided, and then select **Archive**.
-
-   You can restore this flag within 30 days.
-
+ 3. The Archive Flag confirmation screen appears next. If the flag you're archiving is a prerequisite to a another flag, you can select the flag from this screen to remove the dependency, and then try archiving again. 
+   
+   ![The Archive Warning Message pop-up](./static/ff-archive-flag-warning-message.png)
+   
+ 4. If you're sure you want to archive the flag, type the name or ID of the flag in the field provided, and then select **Archive**. Be aware you can restore the archived flag within 30 days.
```
Comment (crystal-fong) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:59]: user only needs to type in the ID of the flag, name's not required 👍

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 2]
```diff
@@ -11,91 +11,89 @@ helpdocs_is_private: false
...
 
 Restoring a flag makes it available again for evaluation if the flag is referenced in your code. That is, you haven't removed the references to it when you archived the flag, or you've reinstated those references.
 
 When you restore a flag, it's restored to all environments. All of the flag's configurations in an environment (including whether it's enabled or disabled) will be restored to the state they were in, in that environment, when the flag was archived. If you created new environments since the flag was archived, the flag will be disabled in those.
 
-To restore a flag:
+Here are the steps to restore an archived flag:
+
+ 1. Select the Harness project containing the flag you'd like to restore. Navigate to **Feature Flags** which will display all of your flags in your project. 
+ 2.
```
Comment (crystal-fong) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:70]: should this 2. be beside `A list of archived flags is displayed` ?

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 3]
```diff
@@ -11,91 +11,89 @@ helpdocs_is_private: false
...
+
+ 1. Select the Harness project containing the flag you'd like to restore. Navigate to **Feature Flags** which will display all of your flags in your project. 
+ 2. 
+
 
-1. In your Harness project, navigate to **Feature Flags > Feature Flags**, and then select the **Archived** filter at the top of the page.
 
    A list of archived flags is displayed.
 
 1. Next to the flag you want to restore, select **more options** (**︙**), and then select **Restore**.
```
Comment (crystal-fong) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:76]: should these be step 3 and 4?

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 4]
```diff
@@ -113,7 +111,3 @@ To delete an archived flag:
 1. Next to the flag you want to delete, select **more options** (**︙**), and then select **Delete**.
 1. On the confirmation screen, select **Delete**.
```
Comment (crystal-fong) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:112]: should this be step 2?

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 5]
```diff
@@ -18,23 +18,74 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
 
 This topic describes the roles available for Feature Flags. For more information about how RBAC works in Harness and instructions for implementing access control, go to [RBAC in Harness](/docs/platform/role-based-access-control/rbac-in-harness).
 
-## Feature Flags roles and permissions
+## Creating & Managing Roles
```
Comment (crystal-fong) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:21]: should the default be and ampersand or `and`, it seems to be a mix of both

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 6]
```diff
@@ -18,23 +18,74 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
 
-The **Feature Flag Manage Role** default Role is available specifically for Feature Flags. On a project of Environment level, this role can:
+The **Feature Flag Admin Role** is a default Role available specifically for Feature Flags. The administrator roles gives you the ability to grant and revoke  permissions at an environment level. This includes:
```
Comment (crystal-fong) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:23]:
```suggestion
The **Feature Flag Admin Role** is a default Role available specifically for Feature Flags. The administrator roles gives you the ability to grant and revoke permissions at an environment level. This includes:
```

Comment (crystal-fong) [reply docs/feature-flags/ff-security-compliance/manage-access-control.md:23]: just want to double check - are we referring to one or more administrator role here?

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 7]
```diff
@@ -18,23 +18,74 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
+
+* Creating, editing and deleting **Resources Groups**,
+* Creating and editing **Default Settings**,
+* Editing and deleting **Projects**,
+* Managing **User Groups**,
+* Creating, editing, managing and deleting **Service Accounts**,
+* Creating, Editing and deleting **Roles**, and
```
Comment (crystal-fong) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:30]:
```suggestion
* Creating, editing and deleting **Roles**, and
```

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 8]
```diff
@@ -18,23 +18,74 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
+To set roles and permissions at the Project level, navigate to the **Project** section from the module navigation bar, and select **Access Control**. Follow similar steps as above to establish the roles and permissions for the project level.
+
+## Creating & Managing Resorce Groups
+
+Here's how you can create and manage resource groups for the Feature flags and Target Management at the account level. Additionally, you can refer to [Manage Resource Groups](/docs/platform/role-based-access-control/add-resource-groups.md) document to learn more.
+
+ ![The Feature Flag Permissions for Resource Groups](../ff-security-compliance/static/ff-resource-groups-permissions.png)
+
+ 1. Navigate to **Account Settings > Access Control >
+ Resource groups within your Harness Account.
```
Comment (crystal-fong) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:77]: does this need a new line? same on line 79

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md (hunk 2 above)
Comment (jenniferopal) [reply docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:70]: Yes, I missed that typo!

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md (hunk 3 above)
Comment (jenniferopal) [reply docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md]: I split it into three steps instead. Hopefully this is better!

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md (hunk 4 above)
Comment (jenniferopal) [reply docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:106]: Yes, it should! Thanks for spotting that!

File: docs/feature-flags/ff-security-compliance/manage-access-control.md (hunk 6 above)
Comment (jenniferopal) [reply docs/feature-flags/ff-security-compliance/manage-access-control.md:23]: This is just the one. I thought this was the only default role and the admin creates the additional roles? 
Is that correct?

Comment (crystal-fong) [reply docs/feature-flags/ff-security-compliance/manage-access-control.md:23]: ya that's right!

File: docs/feature-flags/ff-security-compliance/roles-and-permissions.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+In line with the custom roles and permissions, Role Based Access Control (RBAC) is a useful tool you can set to have more control over your workflow. If you'd like to learn more about [RBAC within the Harness Platform](../../platform/role-based-access-control/rbac-in-harness.md), do have a look at our documentation to learn more.
+:::
+
+## Roles
+
+ ![The Feature Flag Admin Role in Harness](./static/ff-feature-flag-admin-role.png)
+
+  ![The Feature Flag Admin Role Control Panel in Harness](./static/ff-admin-role-access-control.png)
+
+Harness Feature Flags includes built-in roles for you to manage RBAC within the module. To examine the permissions assigned to these roles, to view them:
```
Comment (Kira-Jennings) [inline docs/feature-flags/ff-security-compliance/roles-and-permissions.md:32]:
```suggestion
Harness Feature Flags includes built-in roles for you to manage RBAC within the module. To examine the permissions assigned to these roles:
```

Comment (Kira-Jennings) [reply docs/feature-flags/ff-security-compliance/roles-and-permissions.md:32]: The last sentence here didn't quite make sense

File: docs/feature-flags/ff-security-compliance/roles-and-permissions.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+ 1. Select **Account Settings**, and then select **Access Control** along the top of the page, or by scrolling down.
```
Comment (Kira-Jennings) [inline docs/feature-flags/ff-security-compliance/roles-and-permissions.md:34]: Should we add that this is at the account level/scope only, the same as 2 & 3 mention scope?

Comment (jenniferopal) [reply docs/feature-flags/ff-security-compliance/roles-and-permissions.md:34]: Yes, of course. I'll clarify that further. Thanks for pointing that out. :)

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 11]
```diff
@@ -18,23 +18,72 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
+ 1. Navigate to **Project Settings**. 
+ 2. Scroll down, or select  **Access Control** along the menu and select **Roles** within your Harness
+ Account.
+ 3. From here, you can add a new role or select an existing one to modify.
+ 4. Within the role, select **Feature Flags**. This action will display the Feature flags and Target Management permissions. 
+
+ ![The Panel for the Feature Flag Role Permissions](../ff-security-compliance/static/ff-feature-flag-role-permissions.png)
+
+Feature flags is governed by the following permissions:
+
```
Comment (HarnessTheChaos) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:54]: Can these be bulleted out on their own line so it's easier to read?

image

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 12]
```diff
@@ -18,23 +18,72 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
+ **Create**: Enables users to create new feature flags.
+ **Edit Config**: Enables users to edit existing configs.
+ **Edit Rules**: Enables users to edit rules.
+ **Toggle**: Enables users to toggle the feature flags on or off.
+ **Delete**: Allows users to delete any targets.
+
+Target Management is governed by the following permissions:
+
```
Comment (HarnessTheChaos) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:62]: Same type of bullets and single lines for ease of reading

File: docs/feature-flags/ff-security-compliance/manage-access-control.md [hunk 13]
```diff
@@ -18,23 +18,72 @@ Harness provides Role-Based Access Control (RBAC) that enables you to control us
...
+## Creating and Managing Resorce Groups
+
+Here's how you can create and manage resource groups for the Feature flags and Target Management at the account level. Additionally, you can refer to [Manage Resource Groups](/docs/platform/role-based-access-control/add-resource-groups.md) document to learn more.
+
+ ![The Feature Flag Permissions for Resource Groups](../ff-security-compliance/static/ff-resource-groups-permissions.png)
+
+ 1. Navigate to **Account Settings > Access Control > Resource groups** within your Harness Account.
+ 2. Add a new resource group or select an existing one to modify.
+ 3. Set the "Resource Scope" accordingly if you are creating one.
+ 4. Within the Resources, select **Feature Flags**, Proxy Keys**, or both if that's what you require.
```
Comment (HarnessTheChaos) [inline docs/feature-flags/ff-security-compliance/manage-access-control.md:79]: Proxy Keys needs the front **

**Review by HarnessTheChaos — CHANGES_REQUESTED** [review]

**Review by HarnessTheChaos — APPROVED** [review]

**Pull request merged**